src/components/App/App.js

```diff
@@ -7,12 +7,11 @@ import { benefits } from '../../utils/config';
 function App() {
   return (
     <>
-      <Button label={'Sample Button'}/>
-      <Section type='benefits' layout='four-columns' title='Benefits'>
-        <CardList cards={benefits}></CardList>
+      <Section type='benefits' layout='horizontal' title='Benefits'>
+        <CardList cards={benefits} type='benefits'/>
       </Section>
-      <Section type='products' layout='vertical' title='Wide choice of transport robots'></Section>
-      <Section type='benefits' layout='horizontal' title='Benefits'></Section>
+      <Section type='products' layout='vertical' title='Wide choice of transport robots'/>
```

Comment:
We may have a merge conflict here, since this line is also in my PR

Reply:
I'll just delete this line and the other superfluous Section tag that I was just using as placeholders.

Reply:
Could you just remove the products section so I can merge it with my code? I need to nest the Carousel in it, so the closing tag is helpful in this particular case

Reply:
I did on a more recent commit. 
![image](https://user-images.githubusercontent.com/37159177/110503184-ee510100-80c9-11eb-91ad-4ea71de6b9a3.png)